concent_api/concent_api/system_check.py

```diff
@@ -129,3 +153,35 @@ def storage_cluster_certificate_path_check(app_configs = None, **kwargs):  # pyl
         elif not os.path.isfile(certificate_path):
             errors.append(create_error_15_ssl_cert_path_is_not_a_file(certificate_path))
     return errors
+
+
+@register()
+def creation_new_chain_segment_time_check(app_configs, **kwargs):  # pylint: disable=unused-argument
+    errors = []
+    if (
+        hasattr(settings, 'PAYMENT_BACKEND') and
+        settings.PAYMENT_BACKEND == 'core.payments.sci_backend'
+    ):
+        creation_new_chain_segment_time = settings.CREATION_NEW_CHAIN_SEGMENT_TIME
```

Comment:
It will still fail if `CREATION_NEW_CHAIN_SEGMENT_TIME` will not exists (as a variable) in settings. You have to check `if hasattr(settings, 'CREATION_NEW_CHAIN_SEGMENT_TIME')` first.

Reply:
Done